.github/CODEOWNERS

```diff
@@ -6,36 +6,23 @@
 * @asaj @cmcewen @timmoreton
 
 # directory and file-level owners. Feel free to add to this!
+/packages/attestation-proxy/ @celo-org/cap
+/packages/attestation-service/ @celo-org/cap
 /packages/azure/ @codyborn
-/packages/blockchain-api/ @jmrossy @cmcewen @i1skn @jeanregisser
 /packages/celotool/ @asaj
-/packages/cli/ @mcortesi @gastonponti @medhak1 @celo-org/devx
-/packages/sdk/ @mcortesi @gastonponti @medhak1 @barbaraliau @celo-org/devx
-/packages/dappkit/ @cmcewen @barbaraliau @celo-org/devx
-/packages/dev-utils/ @mcortesi
+/packages/cli/ @celo-org/applications
+/packages/dev-utils/ @celo-org/applications
 /packages/docs/ @timmoreton @barbaraliau @critesjosh 
-/packages/faucet/ @eelanagaraj @celo-org/devx
-/packages/mobile/ @celo-org/wallet 
-/packages/moonpay-auth/ @annakaz @i1skn 
-/packages/notification-service/ @annakaz @cmcewen @i1skn @jeanregisser
+/packages/env-tests @celo-org/cap
+/packages/faucet/ @eelanagaraj @celo-org/applications
```

Comment:
@eelanagaraj do you still want to be on here? totally fine but want to make sure you don't feel obligated given your new circle!

Reply:
I don't really mind! I doubt there will be frequent reviews for it anyways and I may have the most context on it at this point? I can remove myself later if it becomes an issue -- thanks for flagging though :)